src/router/Router.tsx

```diff
@@ -1,5 +1,25 @@
-import { createBrowserRouter } from "react-router-dom";
-
-const router = createBrowserRouter([{}]);
+import { createBrowserRouter } from 'react-router-dom';
+import Home from '../pages/home/Home';
+import ProductList from '../pages/productList/ProductList';
+import ProductDetail from '../pages/productDetail/ProductDetail';
```

Comment:
희선님 path alias 설정 해주셔서 이렇게 변경해도 괜찮겠네요!
```suggestion
import Home from '@pages/home/Home';
import ProductList from '@pages/productList/ProductList';
import ProductDetail from '@pages/productDetail/ProductDetail';
```

Reply:
좋습니다아~

Reply:
넵넵